perception/lidar_obstacle_detection/baidu_cnn/onnx_bcnn/definition.yaml

```diff
@@ -0,0 +1,27 @@
+version: 1
+enable_testing: false
+network:
+  filename: ./model_files/bcnn.onnx
+  framework: ONNX
+  provenance: https://github.com/kosuke55/train_baiducnn
+  training: Nuscenes dataset v1.0-mini
+  model_license: Apache-2
```

Comment:
Please use "Apache-2.0" as in the official [SPDX identifier](https://spdx.org/licenses/).

Reply:
I updated it. 

by the way, pointpillars using `Apache-2`, is it ok?
https://github.com/autowarefoundation/modelzoo/blob/477b8e1f3f6bb150603a4a338130425a80155ea9/perception/lidar_obstacle_detection/point_pillars_pfe/onnx_fp32_kitti/definition.yaml#L8

Reply:
I fixed it in https://github.com/autowarefoundation/modelzoo/pull/13